src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends AbstractFigure {
+    private final Color color;
```

Comment:
this field is being duplicated in all figures, so it should be moved to the parent Figure abstract class
```suggestion
```